turingschool/savile
# Assets page

#### What does this PR do?

Adds an assets page to the Savile docs as a reference for asset files hosted in our S3 bucket on http://assets.turing.io/

#### Where should the reviewer start?
New `_data/assets.yml` file.

#### How should this be manually tested?
Confirm that all asset files on https://turing-savile-assets.s3.amazonaws.com/?list-type=2 are listed in the assets page.

#### Any background context you want to provide?
No

#### What are the relevant tickets?
[Add assets page and display all assets with URLs](https://3.basecamp.com/3494409/buckets/19192671/todos/3634997064)

#### Screenshots (if appropriate)
<img width="1552" alt="Screen Shot 2021-04-06 at 11 35 57 AM" src="https://user-images.githubusercontent.com/709100/113754798-d7073280-96cc-11eb-986b-211fa747182c.png">
<img width="1552" alt="Screen Shot 2021-04-06 at 11 36 00 AM" src="https://user-images.githubusercontent.com/709100/113754807-d8d0f600-96cc-11eb-8746-e6a4024ca84d.png">


#### Any other deploy steps?
No
